src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellItemRenderer.cs

```diff
@@ -433,6 +433,7 @@ protected override void OnShellSectionPropertyChanged(object sender, PropertyCha
 
 		protected virtual void OnTabReselected(ShellSection shellSection)
 		{
+			_ = shellSection.Navigation.PopToRootAsync();
```

Comment:
Yea, I forgot this wasn't the default behavior on `Android` 

Need to think about this one, I think ideally we'd expose some sort of API out to the user so they could act on this if they want to or not..

Or maybe an API like "NavigateBackToHomeOnReselect"

Reply:
I think it is a good idea. Maybe something like this:
```
xmlns:android="clr namespace:Microsoft.Maui.Controls.PlatformConfiguration.AndroidSpecific;assembly=Microsoft.Maui.Controls"
android:TabbedPage.IsNavigateBackToHomeOnReselectEnabled="True"
```